client/src/plus/collections/v2/collection.tsx

```diff
@@ -23,21 +28,30 @@ export default function CollectionComponent() {
     mutate,
   } = useItems(collectionId);
 
+  const navigationType = useNavigationType();
+  useEffect(() => {
+    if (navigationType === "PUSH") document.documentElement.scrollTo(0, 0);
+  }, [navigationType]);
+
   return collection ? (
     <>
-      <header className="plus-header">
+      <header>
         <Container>
-          <Link to="../">Exit Collection</Link>
+          <Link to="../" className="exit">
+            &larr; Exit
```

Comment:
Having looked at the finished product I think 'Back' might be more appropriate here rather than 'Exit'

Reply:
I was thinking the same, but thought it might be part of the design spec.

Reply:
The final copy never really went through any iteration. I think 'Back' is standard